PowerEditor/installer/APIs/cs.xml

```diff
@@ -16,8 +17,205 @@
+			</Overload>
+			<Overload retVal="void">
+				<Param name="string format, object arg0" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="string format, object arg0, object arg1" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="string format, object arg0, object arg1, object arg2" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="bool value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="char value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="char[] buffer" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="decimal value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="double value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="float value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="int value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="long value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="object value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="string value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="ulong value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="uint value" />
+			</Overload>
+		</KeyWord>
+		<KeyWord name="Console.WriteLine" func="yes">
+			<Overload retVal="void" >
+				<Param name="..." />
+			</Overload>
+			<Overload retVal="void" >
+				<Param name="char[] buffer, int index, int count" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="string format, params object[] arg"/>
+			</Overload>
+			<Overload retVal="void">
+				<Param name="string format, object arg0" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="string format, object arg0, object arg1" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="string format, object arg0, object arg1, object arg2" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="bool value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="char value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="char[] buffer" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="decimal value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="double value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="float value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="int value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="long value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="object value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="string value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="ulong value" />
+			</Overload>
+			<Overload retVal="void">
+				<Param name="uint value" />
+			</Overload>
+		</KeyWord>
+		<keyWord name="Console.Read" func="yes">

```

Comment:
The reason alot of these are not getting suggested is because you used `keyWord` instead of `KeyWord`

Reply:
Ohhh. I will fix that one too.